README.md

```diff
@@ -26,6 +26,18 @@ Project will work on both python 2 and python 3
     buy_order = my_trader.place_buy_order(stock_instrument, 1)
     sell_order = my_trader.place_sell_order(stock_instrument, 1)
 
+    # save the token so you don't need user/password again
+    # should probably encrypt this if saving to a file
+    # or use an environment variable: https://docs.python.org/3/library/os.html
+    token = my_trader.token
+    with open('mytoken.pleasesecurethis', 'w') as f:
+        f.write(token)
+
+    # load from token rather than login
+    with open('mytoken.pleasesecurethis') as f:
+      my_trader.token = f.read()
+
```

Comment:
I believe there is a better, more secure way to do this. Storing the token as plaintext in a file isn't an elegant solution. I definitely like your other changes. 

Any ideas?

Reply:
loading from `env` might be okay, but rando flat-file is kind of a mess.

Reply:
Storing in a plain text file being a bad idea is why I named the file as I did. I think leaving how to store the token up to the user, while informing them it isn't a good idea to store insecurely, is sufficient.

I agree that an environment variable is better, depending on use case. Since this is a library rather than a full application, I think it best to provide examples of both methods, and leave it to the developers using the library.

Adding configparser or KeyRing seems something best handled by the developer using the library, at their discretion. This also doesn't add requirements.